.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Lint and Test code
+
+on:
+  pull_request:
+  schedule:
+    # run once a week on early monday mornings
+    - cron: '22 2 * * 1'
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    env:
+      DJANGO_SETTINGS_MODULE: config.settings.test
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: dear_petition
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/**.txt') }}
```

Comment:
Seeing all the requirements files getting hashed made me curious.. Will the `pytest` environment test with `local` requirements or `production` requirements? If it's only testing using only one of these requrirements files, is this considered a gap in our testing infra?

Reply:
Currently, only `local.txt` is installed below, but the cache is invalidated even if a production requirement changes. I suppose we could install the production requirements too below. Seems like a good idea to me.